### PR TITLE
Modernize code

### PR DESCRIPTION
**Pull request by eumiro**
1. Remove useless default encoding for source files (UTF-8 is the default)
2. Simplify loops and conditions

### CODE REVIEW

File: art/__init__.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```
Comment (sepandhaghighi) [inline art/__init__.py:1]: Would you please revert this default encoding?
We need this one for some special cases.

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@eumiro Well done 💯 

Please take a look at my comment about **default encoding** and also:

1. Update `CHANGELOG.md`
2. Update `AUTHORS.md`

**Comment by eumiro** [comment]
Thank you for the feedback, @sepandhaghighi. Dropped the commit with the `coding` removal and updated `AUTHORS.md`. The other commit is a pure refactoring, so it does not really make sense to add it to `CHANGELOG.md`. It does not change any functionality of the code.

**Review by sadrasabouri — APPROVED** [review]
LGTM

**Review by sepandhaghighi — APPROVED** [review]
🚀

**Pull request merged**